src/Http/Controllers/Controller.php

```diff
@@ -167,7 +167,7 @@ public function getContentBasedOnType(Request $request, $slug, $row)
                     $filesPath = [];
                     foreach ($files as $key => $file) {
                         $filename = Str::random(20);
-                        $path = $slug.'/'.date('F').date('Y').'/';
+                        $path = $slug.'/public/'.date('F').date('Y').'/';
```

Comment:
Wouldn't you want `public/` before `$slug`, not after?

Reply:
oOps. I'm sorry. Fixed

Reply:
Thanks!